src/main/java/ch/njol/skript/conditions/CondIsValidCommand.java

```diff
@@ -0,0 +1,72 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.conditions;
+
+import static ch.njol.skript.command.Commands.scriptCommandExists;
+
+import org.bukkit.Bukkit;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+@Name("Is Valid Command")
+@Description("Checks whether a command/string is a valid plugin command, or a custom Skript command.")
+@Examples({
+	"on command:",
+		"\tif command is a valid command:",
+			"\t\tsend \"%command% is a valid plugin command!\" to executor",
+		"\tif command is a skript command:",
+			"\t\tsend \"%command% is a custom Skript command!\" to executor",
```

Comment:
`custom` is not in the pattern you wrote below

Reply:
`custom` refers to this is a user custom registered Skript command, just for the example `send` effect used